packages/fx-core/templates/core/v3Migration/csharp.app.yml

```diff
@@ -97,4 +99,4 @@ configureApp:
       appPackagePath: ./build/appPackage/appPackage.$\{{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
     # Output: following environment variable will be persisted in current environment's .env file.
     # TEAMS_APP_ID: the id of Teams app
-projectId: {{placeholderMappings.[projectId]}}
\ No newline at end of file
+projectId: {{projectId}}
```

Comment:
can we put the projectId to the beginning of the yml file (together with environmentFolderPath)?

Reply:
sure. good idea